ci/ci/build.py

```diff
@@ -143,6 +143,13 @@ def build(self, batch, code, scope):
             if step.can_run_in_scope(scope):
                 step.cleanup(batch, scope, parent_jobs)
 
+    def deployed_services(self) -> Dict[str, List[str]]:
```

Comment:
When would you have multiple namespaces with deployed services in the build configuration?

Reply:
build.yaml technically permits multiple `CreateNamespace` steps, but I can't imagine us using that anytime soon. I'll make this stricter and leave a comment explaining the assertion.